include/gz/math/SphericalCoordinates.hh

```diff
@@ -136,15 +158,66 @@ namespace gz
       /// \param[in] _latB Latitude of point B.
       /// \param[in] _lonB Longitude of point B.
       /// \return Distance in meters.
-      public: static double Distance(const gz::math::Angle &_latA,
+      /// \deprecated Use DistanceWGS84 instead.
+      public: GZ_DEPRECATED(7) static double Distance(
```

Comment:
Can you add this to the [Migration.md](https://github.com/gazebosim/gz-math/blob/ign-math6/Migration.md)?

Reply:
Done here : https://github.com/gazebosim/gz-math/pull/434/commits/b1baee53f003c614bfeec7c28afb2458a82cd7bc